src/main/java/hudson/plugins/parameterizedtrigger/BuildTrigger.java

```diff
@@ -70,9 +88,11 @@ public void buildDependencyGraph(AbstractProject owner, DependencyGraph graph) {
 		// Can only add dependencies in Hudson 1.341 or higher
 		if (!canDeclare(owner)) return;
 
-		for (BuildTriggerConfig config : configs)
-			for (AbstractProject project : config.getProjectList(owner.getParent(),null))
+		for (BuildTriggerConfig config : configs) {
+			List<AbstractProject> projectList = Util.filter(config.getProjectList(owner.getParent(), null), AbstractProject.class);
```

Comment:
@amuniz Thanks for the feedback.  Workflow jobs are explicitly fired via the BuildTrigger.perform method; they don't participate in the DependencyGraph mechanism (core APIs don't support WorkflowJobs -- it's been raised as a potential enhancement, but not yet).